doc/usage.rst

```diff
@@ -83,6 +83,44 @@ Setting up your target to use a header-only version of ``fmt`` is equally easy::
 
    target_link_libraries(<your-target> PRIVATE fmt::fmt-header-only)
 
+Usage with build2
+=================
+
+
+You can use `build2 <https://build2.org>`_, a dependency manager and a build-system combined, to use ``fmt``:
```

Comment:
Here and elsewhere: try to fit into 80-character limit for readability and consistency with the rest of the docs.

Also `:` -> `.`